interface/new/new_comprehensive.php

```diff
@@ -857,11 +857,11 @@ function end_group()
     $('#create').click(function() { check()});
 
     var check = function(e) {
+        var f = document.forms[0];
```

Comment:
while this will fix the immediate error however, when you click Create New Patient afterwards(dialog cancel) form will force submit anyway where you'd expect a new dup dialog. Caused by a force_submit flag set by default in new_search_popup.php. That logic should go away.
Also, appears it's not catching duplicates on popup open!

Reply:
also remember that dialog is opened inside a submit validation routine where returning false from open will prevent default submit.

Reply:
> the immediate error however, when you click Create New Patient afterwards(dialog cancel) form will force submit anyway where you'd expect a new dup dialog. Caused by a force_submit flag set by default in new_search_popup.php. That logic should go away.
> Also, appears it's not catching duplicates on popup open!

Ok, I thought the force submit was expected behavior as there was a comment mentioning it just above the method call. Thanks, for the review I will do the needed changes.

Reply:
That's from old workflow where dup popup would set the confirm button in new patient form and submit took place on that event after user returned and clicked confirm. We want user to actually confirm in popup or select new patient to go fetch. A cancel would mean I want to go back to what I was doing.

Reply:
I have removed the flag and duplicate check is working as expected.

Reply:
Thanks, will bring in after tarvis. Nice work.